src/components/CoinRow/CoinRow.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
-import { LI, P, IMG, Block } from './CoinRow.styles';
+import CoinHeader from 'components/CoinHeader';
+import { StyledLink } from './CoinRow.styles';
 
 const CoinRow = (props) => {
   return (
-    <LI>
-      <P>{props.market_cap_rank}</P>
-      <Block>
-        <IMG image={props.image} />
-        <P>{props.name}</P>
-        <P>({props.symbol.toUpperCase()})</P>
-      </Block>
-      <P>${props.current_price}</P>
-      <P>{props.price_change_percentage_1h_in_currency}%</P>
-      <P>{props.price_change_percentage_24h_in_currency}%</P>
-      <P>{props.price_change_percentage_7d_in_currency}%</P>
-      <P>${props.total_volume}</P>
-      <P>${props.market_cap}</P>
-      <P>Graph Coming Soon</P>
-    </LI>
+    <StyledLink to={`/coins/${props.id}`}>
+      <p>{props.market_cap_rank}</p>
+      <CoinHeader name={props.name} image={props.image} symbol={props.symbol} />
+      <p>${props.current_price}</p>
+      <p>{props.price_change_percentage_1h_in_currency}%</p>
+      <p>{props.price_change_percentage_24h_in_currency}%</p>
+      <p>{props.price_change_percentage_7d_in_currency}%</p>
+      <p>${props.total_volume}</p>
+      <p>${props.market_cap}</p>
+      <p>Graph Coming Soon</p>
+    </StyledLink>
   );
 };
 
```

Comment:
```
const` CoinRow = ({id, market_cap_rank, name, image, symbol, current_price, ...rest}) => {
  return (
    <StyledLink to={`/coins/${id}`}>
      <p>{market_cap_rank}</p>
      <CoinHeader name={name} image={image} symbol={symbol} />
      <p>${current_price}</p>
      {Object.values(rest).map(value => <div>{value}</div>)
      <p>Graph Coming Soon</p>
    </StyledLink>
  );
};
```

Reply:
I can't do it like this because some values have a '$' appeneded to them, and others have a '%' appended to them.